app/settings/account/-components/opt-out/component.ts

```diff
@@ -0,0 +1,43 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import CurrentUserService from 'ember-osf-web/services/current-user';
+import { restartableTask } from 'ember-concurrency';
+import { waitFor } from '@ember/test-waiters';
+import config from 'ember-get-config';
+import IntlService from 'ember-intl/services/intl';
+import captureException, { getApiErrorMessage } from 'ember-osf-web/utils/capture-exception';
+
+const { support: { supportEmail } } = config;
+
+export default class OptOutComponent extends Component {
+    @service currentUser!: CurrentUserService;
+    @service intl!: IntlService;
+    @service toast!: Toastr;
+
+    get allowIndexingIsFalse() {
+        // allowIndexing is null by default
+        return this.currentUser.user?.allowIndexing === false;
+    }
+
+    indexingPreference = this.currentUser.user?.allowIndexing;
```

Comment:
Should we make `indexingPreference` a `tracked` property and only initialize it in the constructor? Don't know which is a better paradigm.

Reply:
I think I prefer the using the constructor as you explained. A little more verbose, but I think it's a bit more clear